source/platform/psp/gu/gu_main.cpp

```diff
@@ -3331,21 +3330,17 @@ void R_RenderScene (void)
 
 	//setfrustum
 	// disabled as well
-	if (r_refdef.fov_x == 90) {
-		// front side is visible
-		VectorAdd(vpn, vright, frustum[0].normal);
-		VectorSubtract(vpn, vright, frustum[1].normal);
 
-		VectorAdd(vpn, vright, frustum[1].normal);
-		VectorSubtract(vpn, vup, frustum[3].normal);
```

Comment:
note in the old code, frustum[1] is set twice, frustum[2] is not set at all, very dangerous! In addition the normals here are not normalized which is not great either. I really don't know why 90 in particular has different treatment anyway other than some ancient pointless optimization.